policyeval/commands.go

```diff
@@ -175,6 +175,89 @@ func (pe *PolicyEvaluator) HandleCommand(ctx context.Context, evt *event.Event)
 			Stringer("policy_event_id", resp.EventID).
 			Msg("Sent ban policy from command")
 		pe.sendSuccessReaction(ctx, evt.ID)
+	case "!remove-ban", "!remove-unban", "!remove-policy":
+		if len(args) < 2 {
+			pe.sendNotice(ctx, "Usage: `!remove-policy <list> <user ID | server name>`")
+			return
+		}
+		list := pe.FindListByShortcode(args[0])
+		if list == nil {
+			pe.sendNotice(ctx, `List %q not found`, args[0])
+			return
+		}
+		target := args[1]
+		var match policylist.Match
+		var entityType policylist.EntityType
+		if !strings.HasPrefix(target, "@") {
+			entityType = policylist.EntityTypeServer
+			match = pe.Store.MatchServer(pe.GetWatchedLists(), target)
+		} else {
+			entityType = policylist.EntityTypeUser
+			match = pe.Store.MatchUser(pe.GetWatchedLists(), id.UserID(target))
+		}
```

Comment:
This probably needs some more checks. If the command is `remove-ban` or `remove-unban`, only allow removing that type of match, not the other kind. Also, there can be wildcard matches, multiple matches and other such things, but handling those is complicated, so maybe just add a todo for those